apps/total-typescript/src/pages/thanks/purchase.tsx

```diff
@@ -110,88 +98,205 @@ const ThanksVerify: React.FC<
+        access your purchase and start learning.
+      </p>
+      <InlineTeamInvite bulkCouponId={bulkCouponId} />
+    </>
+  )
+}
+
+const ExistingBulkCouponPostPurchasePage: React.FC<
+  React.PropsWithChildren<{
+    product: SanityProduct
+    seatsPurchased: number
+    bulkCouponId: string
+  }>
+> = ({product, seatsPurchased, bulkCouponId}) => {
+  return (
+    <>
+      <h1 className="max-w-md text-lg font-medium text-cyan-100 sm:text-lg">
+        Thank you for purchasing more seats for{' '}
+        {product?.name || process.env.NEXT_PUBLIC_SITE_TITLE}!{' '}
+        <br className="hidden sm:block" />
+        Your purchase is for <strong>{seatsPurchased}</strong> additional seat
+        {seatsPurchased > 1 && 's'}.
+      </h1>
+      <InlineTeamInvite bulkCouponId={bulkCouponId} />
+    </>
+  )
+}
+
+const IndividualToBulkPostPurchasePage: React.FC<
+  React.PropsWithChildren<{
+    product: SanityProduct
+    seatsPurchased: number
+    bulkCouponId: string
+  }>
+> = ({product, seatsPurchased, bulkCouponId}) => {
+  return (
+    <>
+      <h1 className="max-w-md text-lg font-medium text-cyan-100 sm:text-lg">
+        Thank you for purchasing more seats for{' '}
+        {product?.name || process.env.NEXT_PUBLIC_SITE_TITLE}!{' '}
+        <br className="hidden sm:block" />
+        Your purchase is for <strong>{seatsPurchased}</strong> additional seat
+        {seatsPurchased > 1 && 's'}. You can always add more seats later when
+        your team grows.
+      </h1>
+      <InlineTeamInvite bulkCouponId={bulkCouponId} />
+    </>
+  )
+}
+
+const ThanksVerify: React.FC<
+  React.PropsWithChildren<{
+    email: string
+    seatsPurchased: number
+    purchaseType: PurchaseType
+    bulkCouponId: string
+    product: SanityProduct
+  }>
+> = ({email, seatsPurchased, purchaseType, bulkCouponId, product}) => {
+  let postPurchasePage = null
+
+  switch (purchaseType) {
+    case NEW_INDIVIDUAL_PURCHASE:
+      postPurchasePage = (
+        <NewIndividualPostPurchasePage product={product} email={email} />
+      )
+      break
+    case NEW_BULK_COUPON:
+      postPurchasePage = (
+        <NewBulkCouponPostPurchasePage
+          product={product}
+          email={email}
+          bulkCouponId={bulkCouponId}
+          seatsPurchased={seatsPurchased}
+        />
+      )
+      break
+    case EXISTING_BULK_COUPON:
+      postPurchasePage = (
+        <ExistingBulkCouponPostPurchasePage
+          product={product}
+          bulkCouponId={bulkCouponId}
+          seatsPurchased={seatsPurchased}
+        />
+      )
+      break
+    case INDIVIDUAL_TO_BULK_UPGRADE:
+      postPurchasePage = (
+        <IndividualToBulkPostPurchasePage
+          product={product}
+          bulkCouponId={bulkCouponId}
+          seatsPurchased={seatsPurchased}
+        />
+      )
+      break
+  }
+
+  return (
+    <PurchaseLayout productImage={product?.image}>
+      {postPurchasePage}
+      {/* link to invoice here! */}
```

Comment:
Link to invoice could be injected here to show up at the bottom of each post-purchase page.